packages/sdk/python/human-protocol-sdk/human_protocol_sdk/job.py

```diff
@@ -364,8 +374,18 @@ def launch(self, pub_key: bytes) -> bool:
         LOG.info("Job's escrow contract deployed to:{}".format(job_addr))
         self.job_contract = get_escrow(job_addr, self.hmt_server_addr)
 
-        (hash_, manifest_url) = upload(self.serialized_manifest, pub_key)
-        self.manifest_url = manifest_url
+        hash_ = self.storage.upload_files(
```

Comment:
instead of hash_ let's call it the style of hash it is. Like is it a manifest hash? A input file hash?

Reply:
@posix4e Everything related to job will be redone in a future PR. These are temporary fixes to make the old SDK version work